apps/OboeTester/app/src/main/java/com/google/sample/oboe/manualtest/StreamConfiguration.java

```diff
@@ -177,11 +178,11 @@ public void setSharingMode(int sharingMode) {
     static String convertSharingModeToText(int sharingMode) {
```

Comment:
Couldn't you use `oboe::convertToText` for this?

Reply:
ditto